service/src/test/java/uk/gov/hmcts/reform/fpl/utils/EmailNotificationHelperTest.java

```diff
@@ -101,7 +100,7 @@ void subjectLineShouldBeSuffixedWithHearingDate() {
         final LocalDateTime dateInTenMonths = LocalDateTime.now().plusMonths(10);
         CaseData caseData = CaseData.builder()
             .respondents1(createRespondents())
-            .hearingDetails(createHearingBookings(dateInTenMonths))
+            .hearingDetails(CaseDataGeneratorHelper.createHearingBookingsFromInitialDate(dateInTenMonths))
```

Comment:
just for consistency, could we have this as static imports?

Reply:
I will fix.